### PR TITLE
LF HelpDesk email is not effective anymore

### PR DESCRIPTION
**Pull request by idvoretskyi**
See https://github.com/kubernetes/community/issues/3502

Signed-off-by: Ihor Dvoretskyi <ihor@linux.com>

### CODE REVIEW

File: contributors/guide/contributor-cheatsheet/README.md [hunk 1]
```diff
@@ -79,7 +79,6 @@ make your GitHub contribution experience better.
 - social@cncf.io - Contact the CNCF social team; blog, twitter account, and other social properties.
 - steering@kubernetes.io - Mail the steering committee. Public address with public archive.
 - steering-private@kubernetes.io - Mail the steering committee privately, for sensitive items.
-- helpdesk@rt.linuxfoundation.org - Mail the LF helpdesk for help with CLA issues.
```
Comment (nikhita) [inline contributors/guide/contributor-cheatsheet/README.md]: Can we instead add info about https://support.linuxfoundation.org?

Comment (nikhita) [reply contributors/guide/contributor-cheatsheet/README.md]: Hmm...it's not an email alias though :/

Comment (mrbobbytables) [reply contributors/guide/contributor-cheatsheet/README.md]: I'll follow up with more explicit instructions on the CLA.

Comment (idvoretskyi) [reply contributors/guide/contributor-cheatsheet/README.md]: @nikhita we can, but not in this section (it's not an email alias anymore).

**Pull request merged**